src/java.base/share/classes/javax/crypto/Cipher.java

```diff
@@ -651,8 +651,8 @@ private String getProviderName() {
      * Returns a {@code Cipher} object that implements the specified
      * transformation.
      *
-     * <p> A new Cipher object encapsulating the
-     * CipherSpi implementation from the specified Provider
+     * <p> A new {@code Cipher} object encapsulating the
+     * {@code CipherSpi} implementation from the specified Provider
```

Comment:
Since `Provider` here is a parameter, it is probably better to put this in an `@code` tag.

Reply:
Yes, I will use the {@code provider} for the "provider" parameter.